### PR TITLE
ubi9: change text to reflect rhcs 5 -> 6

### PR DESCRIPTION
**Pull request by bigjust**
Signed-off-by: Justin Caratzas <jcaratza@redhat.com>

Description of your changes:

Which issue is resolved by this Pull Request:
Resolves #

Checklist:
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.

### CODE REVIEW

**Comment by ktdreyer** [comment]
Want to edit `contrib/rhcs.md` as well?

**Review by guits — APPROVED** [review]

File: contrib/compose-rhcs.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ set -e
 STAGING_DIR=staging/main-ubi9-latest-x86_64/
 DAEMON_BASE_DIR=$STAGING_DIR/daemon-base/
 DOCKERFILE_DAEMON_BASE=$DAEMON_BASE_DIR/Dockerfile
-COMPOSED_DIR=$STAGING_DIR/composed
+COMPOSED_DIR=$STAGING_DIRcomposed
```
Comment (guits) [inline contrib/compose-rhcs.sh:12]:
```suggestion
COMPOSED_DIR="${STAGING_DIR}"composed
```

**Review by guits — CHANGES_REQUESTED** [review]

**Comment by guits** [comment]
jenkins test ceph_ansible-all_daemons

**Comment by guits** [comment]
jenkins test ceph_ansible-collocation

**Comment by guits** [comment]
jenkins test ceph_ansible-lvm_osds

**Review by guits — APPROVED** [review]

**Pull request merged**